.github/workflows/ci.yml

```diff
@@ -44,14 +44,14 @@ jobs:
           - version: 14
 
         pair:
-          - elixir: 1.11.4
-            otp: 23.3.3
+          - elixir: 1.17.2
+            otp: 25.3
```

Comment:
```suggestion
          - elixir: 1.11.4
            otp: 23.3.3
```

Reply:
Ah, I made this change because the test won't pass below 1.17.2